toolchain/sem_ir/inst_namer.h

```diff
@@ -164,6 +173,7 @@ class InstNamer {
   std::vector<std::pair<ScopeId, Namespace::Name>> insts;
   std::vector<std::pair<ScopeId, Namespace::Name>> labels;
   std::vector<Scope> scopes;
+  std::vector<ScopeId> generic_scopes;
```

Comment:
Should this have a `_` suffix? The others are also missing one, I remember thinking about this during recent formatter changes and I'd meant to clean it up, but it slipped my mind. I understand if you don't want to rename the existing ones in this PR, but other fields already have a suffix `_` so it doesn't strike me as simple consistency.

Also, should it have comments? I think it might help newcomers understand the behavior.

Reply:
Added comments and trailing `_`s to all these members.